auto-entrenamiento.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Competencia Peruana de Informática Online
+description: La CPIO es una competencia que a nivel nacional que convoca a estudiantes pre-universitarios con habilidades de solución de problemas. Los estudiantes hacen uso de la lógica, su ingenio y habilidades de programación para resolver los desafíos presentados.
+subtitle: Por el talento peruano;
+---
+
+# Material para entrenamiento
+
+Los interesados a participar en los procesos de selección de la Federación Olímpica Peruana de Informática pueden usar el siguiente material de referencia para prepararse:
+
+**Nota:** Considere el [Sílabo de la IOI](https://people.ksp.sk/~misof/ioi-syllabus/) como principales temas a evaluar.
+
+*Los materiales de referencia a continuación tendrán especificado el idioma si este no está en inglés*
+
+## Teoría
+
+### Libros principales
+
+- Cormen, T. H., Leiserson, C. E., Rivest, R. L., & Stein, C. (2009). Introduction to algorithms. MIT press.
```

Comment:
Algorithms Unlocked? Creo que Introduction to Algorithms es muy pesado para ir primero. Incluso Algorithms Design es más light.

Reply:
Yo creo que el Cormen no sirve de mucho. En mi opinion el mas util es el de Tardos porque te enseña el camino mental para resolver problemas.